packages/backend/src/Application.ts

```diff
@@ -47,6 +47,13 @@ export class Application {
     )
 
     const livenessModule = createLivenessModule(config, logger, database, clock)
```

Comment:
why do we need this module? can it be moved to trackedTxsModule?

Reply:
Well, we are creating the updater and router inside